swift/ql/test/library-tests/dataflow/flowsources/FlowSourcesInline.ql

```diff
@@ -35,10 +35,10 @@ module FlowSourcesTest implements TestSig {
       value = describe(source)
     )
     or
-    exists(DataFlow::Node source, DataFlow::Node sink |
+    exists(DataFlow::Node sink |
       // this is not really what the "flowsources" test is about, but sometimes it's helpful to
       // have sinks and confirm that taint reaches obvious points in the flow source test code.
-      TestFlow::flow(source, sink) and
+      TestFlow::flow(_, sink) and
```

Comment:
This doesn't _really_ matter in a test, but there's room for a performance optimization here: every time we write `flow(_, sink)` we should replace it with `flowTo(sink)` since the latter removes a call to the expensive `fastTC` HOP.

Reply:
Yes, I forget that it isn't just implemented as `flow(_, sink)` any more.  It's good to get into the habit of using it.  Updated.